app/src/main/java/org/covidwatch/android/ui/menu/MenuItem.kt

```diff
@@ -2,7 +2,7 @@ package org.covidwatch.android.ui.menu
 
 data class MenuItem(
     val title: Int,
-    val iconEnd: Int,
+    var iconEnd: Int,
```

Comment:
If we change the items from the `MenuFragment` this property can be `val` again. Then, in the adapter you could create two instance of the `TestResults` menu items one with icon and one without, and in the `MenuFragment` you would call method in `MenuAdapter` to change the the items in the menu item list.